src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -284,11 +354,20 @@ public PerUnitTargetMetricCollector(final String probeSetName, final Set<Interva
             metrics.GENOME_SIZE      = genomeSize;
 
             this.coverageByTarget = new LinkedHashMap<Interval, Coverage>(coverageTargets.size() * 2, 0.5f);
-            for (Interval target : coverageTargets) {
+            for (final Interval target : coverageTargets) {
                 this.coverageByTarget.put(target, new Coverage(target,0));
             }
 
+            this.mapQFilter = new CountingMapQFilter(minimumMappingQuality);
```

Comment:
Slightly confusing that this is declared out-of-order from it's adding to the filter list.  Could you move it so that the creation and add are adjacent?